src/v3/src/components/Images/AppIcon.tsx

```diff
@@ -10,20 +10,24 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { Theme } from '@mui/material';
+import { useOdysseyDesignTokens } from '@okta/odyssey-react-mui';
 import { AppsIcon } from '@okta/odyssey-react-mui/icons';
 import { FunctionComponent, h } from 'preact';
 
 import { loc } from '../../util';
 
-export const AppIcon: FunctionComponent = () => (
-  <AppsIcon
-    sx={(theme: Theme) => ({
-      color: theme.palette.primary.main,
-      width: '16px',
-      height: '16px',
-    })}
-    titleAccess={loc('icon.title.application', 'login')}
-    aria-hidden
-  />
-);
+export const AppIcon: FunctionComponent = () => {
+  const Tokens = useOdysseyDesignTokens();
+
+  return (
+    <AppsIcon
+      sx={{
+        color: Tokens.PalettePrimaryMain,
+        width: '16px',
+        height: '16px',
+      }}
+      titleAccess={loc('icon.title.application', 'login')}
+      aria-hidden
+    />
+  );
+};
```

Comment:
Updated custom icons that Denys added to use token overrides instead of `theme.palette`. We're trying to remove all uses of `theme` in favor of tokens